openslides/users/views.py

```diff
@@ -159,12 +159,17 @@ def reset_password(self, request, pk=None):
         """
         View to reset the password using the requested password.
         """
-        user = self.get_object()
-        if request.data.get('password'):
-            user.default_password = request.data['password']
-        user.set_password(user.default_password)
-        user.save()
-        return Response({'detail': _('Password successfully reset.')})
+        # check manager perms
+        if request.user.has_perm('users.can_manage'):
+            user = self.get_object()
+            if isinstance(request.data.get('password'), str):
+                user.set_password(request.data.get('password'))
```

Comment:
I still do not understand why the default password is immutable. Why can't we use the new set password dialog to set a new password and also to set the initial password to this value. Then the admin may print a user access data sheet if he likes. @emanuelschuetze ?